sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-helper.ts

```diff
@@ -58,12 +77,15 @@ export async function createInstitutionLocations(
  * @param institution institution.
  * @param institutionCode institution code.
  * @param e2eDataSources e2e data sources.
+ * @param options options.
+ * - `initialValues` initial values of institution location.
  * @returns institution location.
  */
 async function findOrCreateInstitutionLocation(
   institution: Institution,
   institutionCode: string,
   e2eDataSources: E2EDataSources,
+  options?: { initialValues?: Partial<InstitutionLocation> },
```

Comment:
The code that already has, has `initialvalue` as a param (which was done as per the suggestion), 
![image](https://github.com/bcgov/SIMS/assets/77353155/14b0cc9a-a25b-48f0-8702-640c26e3f52f)
Maybe we can finalize either one

Reply:
@ann-aot I comment about it here https://github.com/bcgov/SIMS/pull/2011#discussion_r1229876459
The idea of having it inside the options is also to not have to call methods providing undefined sometimes. The idea is to follow this, assuming that it would also make the development easier unless someone sees some drawback.